demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -136,6 +276,27 @@ def validate_json_when_breaking_changes(self) -> bool:
                     is_valid = False
         return is_valid
 
+    @error_codes('RN113,RN114')
```

Comment:
we add a decorator of multiple error codes if all of them are checked in the method directly. this is not the case - then we can add the decorator for each function separately

Reply:
The issue is that both of the functions need the pre-prepare extract and filter so how can we split it without duplicating some calls?

Reply:
shouldn't be an issue, please consult with @MLainer1 about it